tests/test_kohya_loras_scaffold.py

```diff
@@ -0,0 +1,347 @@
+#
+#
+# TODO: REMOVE THIS FILE
```

Comment:
We add this approach to Diffusers or not but I would like to thank you for adding it in this PR anyway. I learned TONS. The design, the thought process, the execution -- everything is very clean and readable. Especially the readability part -- I understand it's a complex problem to solve -- yet the code was fairly readable and simple to handle it. 

Really, thanks! I learned many things from it.